app/assets/v2/js/pages/bounty_details.js

```diff
@@ -1051,27 +1067,44 @@ const build_uri_for_pull_bounty_from_api = function() {
   return uri;
 };
 
+const get_bounty_by_id = function(bountyId) {
+  const deferred = $.Deferred();
+
+  return $.get('/actions/api/v0.1/bounties/' + document.bountyId)
+    .then(result => deferred.resolve([result]));
+};
+
+const get_bounties = function() {
```

Comment:
maybe call it `get_bounty` instead of `get_bounties` since it'lll really only ever be expected to return one?